### PR TITLE
Enable NTP to keep system clock sync

### PR DESCRIPTION
**Pull request by dsimog01**
System clock being out of sync is causing issues, as some clients are throwing in errors during the EL and CL connection as shown in this Lighthouse log:

```
Aug 30 19:14:11.002 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/225517, status: 503 Service Unavailable, elapsed: 426.093µs
```

In this PR, NTP is installed, enabled on boot and started to keep the system clock synced

### CODE REVIEW

File: src/scripts/enable_ntp.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Reference:
+##    https://wiki.debian.org/NTP
+
+# Install ntp package
+apt-get update
+apt-get install -y ntp
+
+# Enable on boot and restart ntp service
+sudo systemctl enable ntp
+sudo systemctl restart ntp
```
Comment (pablomendezroyo) [inline src/scripts/enable_ntp.sh:14]: Is there any posibility to only restart the ntp service to those having the issue?

Comment (dsimog01) [reply src/scripts/enable_ntp.sh:14]: I'll look into it

**Review by 3alpha — APPROVED** [review]

**Comment by Marketen** [comment]
@dsimog01 There seems to be a better alternative to NTP: chrony. Some users reported better results with it, and it is recommended by some staking-related pages: https://ethstaker.gitbook.io/ethstaker-knowledge-base/help/missed-attestations. I have also seen that time desync issues not only prevent sycning with the chain, but also attestation misses.

nflaig comment: https://discord.com/channels/747647430450741309/1143963401140318308/1149622288019378230

**Comment by nflaig** [comment]
> I have also seen that time desync issues not only prevent sycning with the chain, but also attestation misses.

One common issue I saw is that the head is always one slot behind. This leads to 100% missed head votes in attestations and a lot of them are missed due to not being  inlcuded in an aggregate which is related to missed head votes.

**Review by 3alpha — APPROVED** [review]

**Comment by dsimog01** [comment]
Here there is a comparison between chrony and NTP: https://chrony-project.org/comparison.html

**Review by pablomendezroyo — APPROVED** [review]

**Pull request merged**